packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.less

```diff
@@ -0,0 +1,14 @@
+.datacenter-backup {
+  section {
+    border-top: 1px solid #bbbdbf;
+    margin: 25px 0;
+  }
+
+  .font-weight-normal .oui-checkbox__label {
+    font-weight: normal;
+  }
+
+  .font-weight-bold .oui-field .oui-label {
```

Comment:
`font-weight-bold` is a bootstrap utility, why do we need to override it ?

Reply:
This is not required to be over-ridden. This is fixed